sql/sql_yacc.yy

```diff
@@ -6606,6 +6606,12 @@ column_def:
           { $$= $1; }
         | field_spec references
           { $$= $1; }
+        | field_spec constraint check_constraint
+          {
+            $$= $1;
+            $$->check_constraint= $3;
+            $$->check_constraint->name= $2;
+          }
```

Comment:
please, rebase your PR to 10.5. I suspect this hunk might be a problem there.

Reply:
Yes, seems there are some problems with 10.5. :hammer_and_pick: